resources/badcert.html

```diff
@@ -3,6 +3,24 @@
 <title>Certificate error</title>
 </head>
 <body>
-    <p>${reason}</p>
+  <p>${reason}</p>
+  <pre id="bytes">${bytes}</pre>
+  <button id="allow">Allow certificate temporarily</button>
```

Comment:
Should there also be a "decline" button?

Reply:
Perhaps. What should it do, however?

Reply:
Firefox does have the Get Me Out Of Here button which I think does the same thing, so I'll add that.